### PR TITLE
feat: Query unbond batches #DRP-195

### PR DESCRIPTION
**Pull request by deniszagumennov**
Task is: https://hadronlabs.atlassian.net/browse/DRP-195

### CODE REVIEW

File: packages/base/src/msg/core.rs [hunk 1]
```diff
@@ -107,6 +107,11 @@ pub enum QueryMsg {
     CurrentUnbondBatch {},
     #[returns(crate::state::core::UnbondBatch)]
     UnbondBatch { batch_id: Uint128 },
+    #[returns(crate::state::core::UnbondBatchesResponse)]
+    UnbondBatches {
+        limit: Option<usize>,
```
Comment (ratik) [inline packages/base/src/msg/core.rs:112]: please replace `usize` by `Uint64`

File: integration_tests/src/testcases/core.test.ts [hunk 2]
```diff
@@ -1274,6 +1274,56 @@ describe('Core', () => {
...
+          status: 'new',
+          status_timestamps: expect.any(Object),
+          expected_release_time: 0,
+          total_dasset_amount_to_withdraw: '0',
+          expected_native_asset_amount: '0',
+          total_unbond_items: 0,
+          unbonded_amount: null,
+          withdrawn_amount: null,
+        });
+      });
```
Comment (albertandrejev) [inline integration_tests/src/testcases/core.test.ts:1326]: can you add test to query all batches at once?

File: packages/base/src/msg/core.rs (hunk 1 above)
Comment (deniszagumennov) [reply packages/base/src/msg/core.rs:112]: Here, in `take` method, we should pass usize:
https://github.com/hadronlabs-org/drop-contracts/blob/4eae28df3698b0d91535d9491d25dcb3abeda7d2/contracts/core/src/contract.rs#L244
like here
https://github.com/hadronlabs-org/drop-contracts/blame/4eae28df3698b0d91535d9491d25dcb3abeda7d2/contracts/auto-withdrawer/src/contract.rs#L277

File: integration_tests/src/testcases/core.test.ts (hunk 2 above)
Comment (deniszagumennov) [reply integration_tests/src/testcases/core.test.ts:1361]: Done

**Review by albertandrejev — APPROVED** [review]

**Pull request merged**